src/components/schedule-visualizer/lift-overlay.tsx

```diff
@@ -0,0 +1,46 @@
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import React, { createContext, useContext } from 'react';
+import { viewBoxFromLeafletBounds } from '../../util/css-utils';
+import SVGOverlay, { SVGOverlayProps } from './svg-overlay';
+import Lift from './lift';
+
+export interface LiftsOverlayProps extends SVGOverlayProps {
+  currentFloor: string;
+  // TODO: this should be replaced by RomiCore.Lift once we addressed this
+  // https://github.com/osrf/romi-js-core-interfaces/issues/4
+  lifts: readonly RomiCoreLift[];
+  onLiftClick?(lift: RomiCoreLift): void;
+}
+
+export default function LiftsOverlay(props: LiftsOverlayProps): React.ReactElement {
+  const { lifts, onLiftClick, currentFloor, ...otherProps } = props;
+  const viewBox = viewBoxFromLeafletBounds(props.bounds);
+  const useStateValue = () => useContext(LiftStateContext);
+  const liftsState = useStateValue();
+  return (
+    <>
+      <SVGOverlay {...otherProps}>
+        <svg viewBox={viewBox}>
+          {lifts.map(lift => (
+            <Lift
+              key={`lift-${lift.name}`}
```

Comment:
Are lift names unique? If that is not the case, this could generate some invalid values for `key`.

Reply:
Let's confirm this with @koonpeng too

Reply:
yes they are unique

Reply:
Great! Thank you!